addons/website/static/src/snippets/s_showcase/002.scss

```diff
@@ -0,0 +1,45 @@
+.s_showcase[data-vcss='002'] {
```

Comment:
As discussed I updated to remove all the scss, and replace with appropriate options.js